.circleci/config.yml

```diff
@@ -19,13 +19,12 @@ jobs:
           name: 'pylint'
           command: |
             source /usr/local/share/virtualenvs/tap-nikabot/bin/activate
-            make lint
-      #TODO: Unit tests failing in circle but pass locally, if contributer helps explain it would be nice to fix these
-      #- run:
-      #    name: 'unit-tests'
-      #    command: |
-      #      source /usr/local/share/virtualenvs/tap-nikabot/bin/activate
-      #      make test
+            pylint tap_nikabot --disable 'broad-except,chained-comparison,empty-docstring,fixme,invalid-name,line-too-long,missing-class-docstring,missing-function-docstring,missing-module-docstring,no-else-raise,no-else-return,too-few-public-methods,too-many-arguments,too-many-branches,too-many-lines,too-many-locals,ungrouped-imports,wrong-spelling-in-comment,wrong-spelling-in-docstring,bad-whitespace'
+      - run:
+          name: 'unit-tests'
+          command: |
+            source /usr/local/share/virtualenvs/tap-nikabot/bin/activate
+            coverage run -m pytest && coverage report
```

Comment:
Is there a reason you're avoiding using the makefile here? I would prefer to reuse the same commands in both places.